pkg/env/node_index.go

```diff
@@ -0,0 +1,26 @@
+package env
+
+import "time"
+
+var (
+	// NodeIndexEnabled defines whether Compliance will actually run indexing code.
+	NodeIndexEnabled = RegisterBooleanSetting("ROX_NODE_INDEX_ENABLED", false)
+
+	// NodeIndexHostPath sets the path where the R/O host node filesystem is mounted to the container.
+	// that should be scanned by Scanners NodeIndexer
+	NodeIndexHostPath = RegisterSetting("ROX_NODE_INDEX_HOST_PATH", WithDefault("/host"))
+
+	// NodeIndexContainerAPI defines the API endpoint for the RepositoryScanner to reach out to.
+	NodeIndexContainerAPI = RegisterSetting("ROX_NODE_INDEX_CONTAINER_API", WithDefault("https://catalog.redhat.com/api/containers/"))
+
+	// NodeIndexMappingURL defines the endpoint for the RepositoryScanner to download mapping information from.
+	NodeIndexMappingURL = RegisterSetting("ROX_NODE_INDEX_MAPPING_URL", WithDefault("https://sensor.stackrox.svc/scanner/definitions?file=repo2cpe"))
+
+	// NodeIndexCacheDuration defines the time a cached node index will be considered fresh and served from file.
+	// Defaults to 75% of the default rescan interval.
+	NodeIndexCacheDuration = registerDurationSetting("ROX_NODE_INDEX_CACHE_DURATION", 3*time.Hour)
```

Comment:
```suggestion
	// Defaults to 75% of the default rescan interval.
	NodeIndexCacheDuration = registerDurationSetting("ROX_NODE_INDEX_CACHE_DURATION", os.Getenv("ROX_NODE_SCANNING_INTERVAL") * 0.75)
```

If we want this to be 3/4 of the node scanning interval perhaps it can be written relatively instead of hardcoded? Not sure if this would work since I don't know when exactly the node scanning interval is set.

Reply:
I'd lean towards leaving it like this to not introduce "magic" settings that depend on each other.
While they are very convenient, we don't have a good way to tell users why 2 settings change when they change 1 env var.